src/lib/y2storage/planned/can_be_formatted.rb

```diff
@@ -58,14 +61,17 @@ def initialize_can_be_formatted
       #
       # Do nothing if #filesystem_type is not set.
       #
+      # FIXME: temporary API. It should be improved.
+      #
       # @param blk_dev [BlkDevice]
       #
       # @return [Filesystems::BlkFilesystem] filesystem
-      def create_filesystem(blk_dev)
+      def format!(blk_dev)
```

Comment:
As already mentioned in this method documentation: temporary API (to be improved while adding LVM support).